### PR TITLE
functionalities to get balance and unlocked balance of vault tokens

### PR DESCRIPTION
**Pull request by jx3yang**
## Description

Add required functionalities for the vault management container. Added a temporary simple table beneath the current geyser container to hook up the functionalities

### CODE REVIEW

File: frontend/src/constants.ts [hunk 1]
```diff
@@ -52,3 +52,10 @@ export const UFRG_INIT_BLOCK = 7947823
 
 export const AMPL_LAUNCH_DATE = 1561687200
 export const INITIAL_SUPPLY = 50000000
+
+// alignment
+export enum Align {
```
Comment (surudhb) [inline frontend/src/constants.ts:57]: very snazzy!

File: frontend/src/styling/styles.ts [hunk 2]
```diff
@@ -6,11 +6,15 @@ export const Paragraph = styled.p`
...
-export const GeyserFirstOverlay = styled.div`
+const Overlay = styled.div`
   ${tw`shadow-all w-full rounded-lg my-2`}
   ${tw`sm:my-4`}
 `
 
+export const GeyserFirstOverlay = styled(Overlay)``
+
+export const VaultFirstOverlay = styled(Overlay)``
+
```
Comment (surudhb) [inline frontend/src/styling/styles.ts]: actually since there is no custom styling, we can just use the `Overlay` you created and purge these two

**Review by surudhb — CHANGES_REQUESTED** [review]
LGTM! Requesting one minor change!

File: frontend/src/styling/styles.ts (hunk 2 above)
Comment (jx3yang) [reply frontend/src/styling/styles.ts]: fixed

**Review by surudhb — APPROVED** [review]
LGTM!

**Pull request merged**